src/github_runner_manager/openstack_cloud/openstack_runner_manager.py

```diff
@@ -736,12 +736,20 @@ def _wait_runner_running(self, instance: OpenstackInstance) -> None:
 
     @staticmethod
     def _generate_instance_id() -> InstanceId:
-        """Generate a instance id.
+        r"""Generate an instance id suffix compliant to the GitHub runner naming convention.
+
+        The GitHub runner name convention is as following:
+        A valid runner name is 64 characters or less in length and does not include '"', '/', ':',
+        '<', '>', '\', '|', '*' and '?'.
+
+        The collision rate calculation:
+        alphanumeric 12 chars long (26 alphabet + 10 digits = 36)
+        36^12 is big enough for our use-case.
```

Comment:
You are changing it to 6 chars, so its `36^6` , isn't it? Could you please update the docstring?

Reply:
It's hex - which = 12 chars